### DISCUSSION TITLE
Some fields in collection cannot be update

### DISCUSSION THREAD

**Discussion by yudyananda** [payloadcms/payload#70] [Q&A] [answered]
Hi i just start payloadcms with [quickstart](https://payloadcms.com/docs/getting-started/installation) on my windows10 (local machine).  Pick the blog template, connect to my local mongo `mongodb://localhost/payloads` then run `yarn dev`

```
$ yarn dev
yarn run v1.22.4
$ nodemon
[nodemon] 2.0.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node server.js`
[03:39:15] INFO (payload): Starting Payload...
[03:39:20] INFO (payload): Payload Admin URL: http://localhost:3000/admin
(node:9384) DeprecationWarning: Listening to events on the Db class has been deprecated and will be removed in the next major version.
[03:39:20] INFO (payload): Connected to Mongo server successfully!
[03:39:23] INFO (payload): E-mail configured with mock configuration
[03:39:23] INFO (payload): Log into mock email provider at https://ethereal.email
[03:39:23] INFO (payload): Mock email account username: XxxaxXXaxax@ethereal.email
[03:39:23] INFO (payload): Mock email account password: XMKXaxanxaAXxaxadada
webpack built f9b9cf5221b841d38de6 in 34192ms

```

I was really impressed by how it looked. Unfortunately I encountered a problem. Some fields in the post collection do not change when edited.

In this case I am trying to change the RichText and Tags fields, click save. Return to the posts collection page, click the content again and the edited field will change back to what it was before editing

I'm not sure if it's a bug, something I missed or something the development edition couldn't do. What do you think?

thanks

**Comment by DanRibbens** [comment, marked as answer]
I was able to recreate the bug also!

![image](https://user-images.githubusercontent.com/6434612/108068858-a1c95700-7030-11eb-9c40-c43c36f0b985.png)

I'm looking in to this issue right now. I assume it has to do with the rich text field. I've opened an issue for the bug here: https://github.com/payloadcms/payload/issues/71

**Reply by yudyananda** [reply]
@DanRibbens  thanks. Can you also reproduce in tag field? i mean try remove existing or add new tag in the existing post content

**Reply by DanRibbens** [reply]
Hey, I forgot to follow up in here since we closed #71. I was able to reproduce with other fields along with richText back when you reported it. We made some refactors around localization on fields that came up since closing #71. We also improved our test coverage related to this as well. 👍 
Get the updated payload version and all should be working if you haven't already.
